fs/fcb/src/fcb.c

```diff
@@ -121,10 +125,29 @@ fcb_free_sector_cnt(struct fcb *fcb)
 int
 fcb_is_empty(struct fcb *fcb)
 {
-    return (fcb->f_active.fe_area == fcb->f_oldest &&
+    return (fcb->f_active.fe_sector == fcb->f_oldest_sec &&
       fcb->f_active.fe_elem_off == sizeof(struct fcb_disk_area));
 }
 
+struct sector_range *
+fcb_get_sector_range(const struct fcb *fcb, int sector)
```

Comment:
This function (`fcb_get_sector_range`) would probably be simpler if it called `fcb_get_sector_info()`.

Reply:
done